File: Natural Language/CalculatePrice.cs
static Dictionary<string, double> itemPrices = new Dictionary<string, double>() 

{ 

    {"apple", 0.5}, 

    {"banana", 0.75}, 

    {"orange", 0.6} 

}; 

 

static double CalculateTotalPrice(string item, int quantity) 

{ 

    double price; 

    if (!itemPrices.TryGetValue(item, out price)) 

    { 

        price = 0; 

    } 

    return quantity * price; 

}

File: Natural Language/CalculatePrice.py
item_prices = { 

    'apple': 0.5, 

    'banana': 0.75, 

    'orange': 0.6 

} 

 

def calculate_total_price(item, quantity): 

    price = item_prices.get(item, 0) 

    return quantity * price
